add-interactive.h

```diff
@@ -0,0 +1,8 @@
+#ifndef ADD_INTERACTIVE_H
+#define ADD_INTERACTIVE_H
+
+int add_i_config(const char *var, const char *value, void *cbdata);
+
+void print_modified(void);

```

Comment:
Since there are no namespaces in C, maybe we could prefix this function name with `add_i_`, too?